openfl-gramine/openfl.manifest.template

```diff
@@ -0,0 +1,80 @@
+loader.preload = "file:{{ gramine.libos }}"
+libos.entrypoint = "/usr/local/bin/python3.8"
+loader.log_level = "{{ log_level }}"
+
+loader.env.LD_LIBRARY_PATH = "{{ python.stdlib }}/lib:/lib:{{ arch_libdir }}:/usr/lib:/usr/{{ arch_libdir }}"
+loader.env.no_proxy = "{{ no_proxy }}"
+loader.env.https_proxy = "{{ https_proxy }}"
+loader.env.http_proxy = "{{ http_proxy }}"
+loader.env.SSL_CERT_DIR = "/etc/ssl/certs"
+
+loader.insecure__use_cmdline_argv = true
+
+sys.enable_sigterm_injection = true
+
+fs.start_dir="/workspace"
+
+
+# .URI - path on host
+# .PATH - pointer inside gramine
+
+fs.mount.lib.type = "chroot"
+fs.mount.lib.path = "/lib"
+fs.mount.lib.uri = "file:{{ gramine.runtimedir() }}"
+
+fs.mount.lib2.type = "chroot"
+fs.mount.lib2.path = "{{ arch_libdir }}"
+fs.mount.lib2.uri = "file:{{ arch_libdir }}"
+
+fs.mount.usr.type = "chroot"
+fs.mount.usr.path = "/usr"
+fs.mount.usr.uri = "file:/usr"
+
+fs.mount.tmp.type = "chroot"
+fs.mount.tmp.path = "/tmp"
+fs.mount.tmp.uri = "file:/tmp"
```

Comment:
I think we could try removing /tmp directory from manifest to see if it works fine

Reply:
I removed mounting `/tmp` and it worked, thank you